pkgs/applications/misc/geoipupdate/default.nix

```diff
@@ -0,0 +1,20 @@
+{ buildGoModule, fetchFromGitHub, stdenv }: buildGoModule rec {
```

Comment:
Let's use this order: `{ stdenv, buildGoModule, fetchFromGitHub }:` and add a newline after `:`.